doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3512,7 +3512,7 @@ Jobs not received after reset
 
 .. rst-class:: v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-5-1 v2-5-0
 
-NCSDK-24305: fota_download library sends FOTA_DOWNLOAD_EVT_FINISHED when unable to connect
+NCSDK-24305: fota_download and fota_download libraries sends FOTA_DOWNLOAD_EVT_FINISHED and FOTA_DOWNLOAD_EVT_FINISHED events, respectively, when unable to connect
```

Comment:
Why 2 entries for the same library and same event? Isn't this about the FOTA download library that is not changed as part of the PR?